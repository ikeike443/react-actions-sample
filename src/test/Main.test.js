import React from 'react';
import Main from '../Main';
import renderer from 'react-test-renderer';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <Main page="http://localhost:3000">Simple SPA</Main>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
