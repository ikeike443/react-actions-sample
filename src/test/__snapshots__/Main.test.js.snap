// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div>
  <h1>
    Simple SPA
  </h1>
  <ul
    className="header"
  >
    <li>
      <a
        aria-current="page"
        className="active"
        href="#/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        className={undefined}
        href="#/stuff"
        onClick={[Function]}
        style={undefined}
      >
        Stuff
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        className={undefined}
        href="#/contact"
        onClick={[Function]}
        style={undefined}
      >
        Contact
      </a>
    </li>
  </ul>
  <div
    className="content"
  >
    <div>
      <h2>
        HELLO
      </h2>
      <p>
        Cras facilisis urna ornare ex volutpat, et convallis erat elementum. Ut aliquam, ipsum vitae gravida suscipit, metus dui bibendum est, eget rhoncus nibh metus nec massa. Maecenas hendrerit laoreet augue nec molestie. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
      </p>
      <p>
        Duis a turpis sed lacus dapibus elementum sed eu lectus.
      </p>
    </div>
  </div>
</div>
`;
